src/app/common/stores/applications/SsvWeb/Wallet.store.ts

```diff
@@ -227,15 +232,15 @@ class WalletStore extends BaseStore implements Wallet {
   /**
    * Fetch user balances and fees
    */
-  async balanceHandler(balance: any) {
+  async onBalanceChangeCallback(balance: any) {
     if (balance) await this.initializeUserInfo();
```

Comment:
I've question, if user has spend all his balance and it will be zero, won't it update his balance?